frontend/src/components/Main/Main.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './Main.scss';
+import { Route, Switch } from 'react-router-dom';
+import Home from '../../pages/Home/Home';
+import Products from '../../pages/Products/Products';
+import AdminOrders from '../../pages/AdminOrders/AdminOrders';
+import AdminProducts from '../../pages/AdminProducts/AdminProducts';
+import CheckOut from '../../pages/CheckOut/CheckOut';
+import Orders from '../../pages/Orders/Orders';
+import OrderSuccess from '../../pages/OrderSuccess/OrderSuccess';
+import ShoppingCart from '../../pages/ShoppingCart/ShoppingCart';
+
+function Main() {
```

Comment:
I would recommend putting all of the string path names into a global constant and replace the uses of those strings with their constant reference

Reply:
Will do